spec/tdd_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../tdd'
+
+describe Solver do
+  solver = Solver.new
+  number = 5
+  three = 3
+  word = 'open'
+
+  context 'Test the solver class factoiral method.' do
+    it 'it should return the factorial of a number passed as an argument' do
+      factorial = solver.factorial(number)
+      expect(factorial).to eql 120
+    end
+  end
+
+  context 'Test the solver class fizzbuzz method' do
+    it 'it should return a string "fizz" when a divisible of 3 is passed as an argument' do
+      string = solver.fizzbuzz(three)
+      expect(string).to eql 'fizz'
+    end
+
+    it 'it should return a string "buzz" when a divisible of 5 is passed as an argument' do
+      string = solver.fizzbuzz(5)
+      expect(string).to eql 'buzz'
+    end
+
+    it 'it should return a the same number as a string when anything other than a divisible of 3 or 5 is passed as an argument' do
+      string = solver.fizzbuzz(7)
+      expect(string).to eql '7'
+    end
+  end

```

Comment:
- [ ]  As required for the project, kindly ensure that The `#fizzbuzz` method has tests that also handle the following case:
  * `Number` is divisible by `3` and `5` with `fizzbuzz` as expectation

Reply:
@tresorsawasawa thank you again for letting me know about the test case that I missed. I will surely add it.

Reply:
Updated [solution](https://github.com/Wahaj-Ali/TDD/pull/1/commits/944cb54f4bde1de41de55594af3caeace39149d4).